.github/workflows/pull_request.yaml

```diff
@@ -58,6 +58,15 @@ jobs:
         if: failure() && steps.ignore_docs.outputs.num_source_files != 0
         timeout-minutes: 30
         run: cd build && ASAN_OPTIONS=fast_unwind_on_malloc=0 ctest --timeout 300 --output-on-failure --rerun-failed
+      - name: Coverage Report
+        if: success() && matrix.compiler == 'gcc' && matrix.tag == 'centos7'
+        run: cd build && lcov --gcov-tool $GCOV --capture --directory . --output-file coverage.info
+      - name: Coveralls
```

Comment:
Could we see the difference result of coverage between PR and master?

Reply:
> Could we see the difference result of coverage between PR and master?

Yes, like this https://github.com/Tencent/rapidjson